src/main/java/com/google/devtools/build/lib/analysis/config/BuildConfiguration.java

```diff
@@ -584,25 +584,6 @@ public StrictDepsConverter() {
     )
     public boolean collectCodeCoverage;
 
-    @Option(
-        name = "incompatible_cc_coverage",
```

Comment:
nit: how about moving this to the option graveyard? There are a few `incompatible_*` options already there.